tools/cli/domainCommands.go

```diff
@@ -469,6 +470,18 @@ func (d *domainCLIImpl) ListDomains(c *cli.Context) {
 
 	domains := d.getAllDomains(c)
 	var filteredDomains []*types.DescribeDomainResponse
+
+	// Only list domains that are matching to the prefix if prefix is provided
+	if len(prefix) > 0 {
```

Comment:
nit: shall we merge this filtering logic with the existing filtering logic below?